webapp/publisher/snaps/views.py

```diff
@@ -677,6 +690,18 @@ def redirect_post_register_name():
 @publisher_snaps.route("/register-snap", methods=["POST"])
 @login_required
 def post_register_name():
+    try:
+        # We don't use the data from this endpoint.
+        # It is mostly used to make sure the user has signed
+        # the terms and conditions.
```

Comment:
Can you give me a use case that would make it fail?
This is handled at login. When the user logins he should be redirected before even getting into this function thanks to the annotation `@login_required`

Reply:
:+1: